packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -136,16 +136,16 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent {
 	public headerClassName?: string;
 
 	/**
-	 * Output that will emit a specific column whenever its filter value has changed
+	 * Output that will emit "void" whenever its filter value has changed
 	 */
 	@Output()
-	public filterChanged: EventEmitter<StarkTableColumnComponent> = new EventEmitter<StarkTableColumnComponent>();
+	public filterChanged: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
In this case, we should adapt the following code:
```typescript
public onFilterChange(): void {
	this.filterChanged.emit();
}
```

And emit **this** instead of void.

Reply:
Indeed ;)